onnxruntime/python/onnxruntime_pybind_quant.cc

```diff
@@ -128,6 +128,8 @@ void QuantizeMatMulBnb4Blockwise(
 void CreateQuantPybindModule(py::module& m) {
   m.def("quantize_matmul_4bits", &QuantizeMatMulNBitsBlockwise<float, 4>);
   m.def("quantize_matmul_4bits", &QuantizeMatMulNBitsBlockwise<MLFloat16, 4>);
+  m.def("quantize_matmul_nbits", &QuantizeMatMulNBitsBlockwise<MLFloat16, 4>);
```

Comment:
>quantize_matmul_nbits [](http://example.com/codeflow?start=9&length=21)

why quantize_matmul_nbits just a dup with quantize_matmul_4bits?
Should it be: m.def("quantize_matmul_2bits", &QuantizeMatMulNBitsBlockwise<MLFloat16, 2>);